win32ss/user/ntuser/kbdlayout.c

```diff
@@ -17,16 +18,60 @@
 
 DBG_DEFAULT_CHANNEL(UserKbdLayout);
 
-PKL gspklBaseLayout = NULL;
+PKL gspklBaseLayout = NULL; // FIXME: Please move this to pWinSta->spklList
 PKBDFILE gpkfList = NULL;
 DWORD gSystemFS = 0;
 UINT gSystemCPCharSet = 0;
 
 typedef PVOID (*PFN_KBDLAYERDESCRIPTOR)(VOID);
 
-
 /* PRIVATE FUNCTIONS ******************************************************/
 
+// Win: _GetKeyboardLayoutList
+static UINT APIENTRY
+IntGetKeyboardLayoutList(PWINSTATION_OBJECT pWinSta, ULONG nBuff, HKL *pHklBuff)
```

Comment:
New code -> please use SAL annotations (and correct formatting)

Reply:
Used SAL. OK.